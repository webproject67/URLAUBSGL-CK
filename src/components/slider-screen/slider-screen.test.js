import React from "react";
import renderer from "react-test-renderer";
import SliderScreen from "./slider-screen";

it(`Render SliderScreen`, () => {
  const tree = renderer.create(<SliderScreen num={2}/>).toJSON();

  expect(tree).toMatchSnapshot();
});