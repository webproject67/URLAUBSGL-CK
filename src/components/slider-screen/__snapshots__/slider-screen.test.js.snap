// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SliderScreen 1`] = `
<div
  className="how-works__item"
>
  <picture>
    <source
      media="(min-width: 960px)"
      srcSet="img/desktop/Layer_2@1x.jpg 1x, img/desktop/Layer_2@2x.jpg 2x"
      type="image/jpg"
    />
    <source
      media="(min-width: 768px)"
      srcSet="img/tablet/Layer_2@1x.jpg 1x, img/tablet/Layer_2@2x.jpg 2x"
      type="image/jpg"
    />
    <source
      media="(max-width: 767px)"
      srcSet="img/mobile/Layer_2@1x.jpg 1x, img/mobile/Layer_2@2x.jpg 2x"
      type="image/jpg"
    />
    <img
      alt="photo"
      className="how-works__img"
      src="img/desktop/Layer_2@1x.jpg"
      srcSet="img/desktop/Layer_2@2x.jpg 2x"
    />
  </picture>
  <a
    className="how-works__link"
    href="#"
  >
    step 1
  </a>
  <h3
    className="how-works__subtitle"
  >
    Sed leo enim, condimentum
  </h3>
  <p
    className="how-works__text"
  >
    Quisque libero libero, dictum non turpis in, luctus semper lorem. Donec rhoncus a leo sit amet facilisis.
  </p>
</div>
`;
